src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("done")) {
+                doneTasks(command);
+                System.out.println("____________________________________________________________");
+            } else if (command.contains("todo")) {
+                System.out.println("____________________________________________________________");
+                todoTask(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("deadline")) {
+                System.out.println("____________________________________________________________");
+                deadlineTasks(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("event")) {
+                System.out.println("____________________________________________________________");
+                eventTask(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("delete")) {
+                System.out.println("____________________________________________________________");
+                deleteTask(command);
+                System.out.println("____________________________________________________________");
+
+            } else {
+                System.out.println("____________________________________________________________");
+                System.out.println("OOPS!!! I'm sorry, but I don't know what that means :-(");
+                System.out.println("____________________________________________________________");
+            }
+        }
+
+
+    }
+
+
+    private static void createDirAndFile() {
+        try {
+            System.out.println("Creating file: data/duke.txt");
+            File newDir = new File("data");
+            newDir.mkdir();
+            File f = new File("data/duke.txt");
+            if (f.createNewFile()) {
+                System.out.println("File created: " + f.getName());
+            } else {
+                System.out.println("File already exists.");
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
+    }
+
+        private static void deadlineTasks(String command) {
+
+        try {
+            String description = command.substring(9, command.indexOf("/by"));
+            String by = command.substring(command.indexOf("/by") + 4);
+            Task task = new Deadline(description, by);
+            tasks.add(task);
+            System.out.println("Got it. I've added this task:");
+            System.out.println(task);
+            System.out.println("Now you have "+ Task.getTaskCount() +" task(s) in the list.");
+            writeToFile("data/duke.txt");
```

Comment:
You have a method `writeToFile` that abstracts the file writing details. Perhaps you can enforce [SLAP](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here.